named_arrays/scalars/tests/test_core.py

```diff
@@ -313,6 +313,28 @@ class TestScalarNormalRandomSample(
 ):
     pass
 
+def _scalar_poisson_random_samples() -> list[na.ScalarNormalRandomSample]:
+    centers = [
+        0,
+        na.ScalarArray(np.random.random(_num_x), axes=('x', )),
+    ]
+    units = [None, u.mm]
+    shapes_random = [dict(y=_num_y)]
+    return [
+        na.ScalarPoissonRandomSample(
+            center=center << unit if unit is not None else center,
+            shape_random=shape_random,
+        ) for center in centers for unit in units for shape_random in shapes_random
+    ]
+
+
+@pytest.mark.parametrize('array', _scalar_normal_random_samples())
```

Comment:
You forgot to call `_scalar_poisson_random_sample()` here. That's why codecov is complaining about the coverage